scripts/release/release-stable.sh

```diff
@@ -29,6 +29,8 @@ This script will release stable ESPv2 docker image with format of:
   $(get_proxy_image_release_name):\${MAJOR_BASE_VERSION}
   $(get_serverless_image_release_name):\${MINOR_BASE_VERSION}
   $(get_serverless_image_release_name):\${MAJOR_BASE_VERSION}
+  $(get_gcsrunner_image_release_name):\${MINOR_BASE_VERSION}
+  $(get_gcsrruner_image_release_name):\${MAJOR_BASE_VERSION}
```

Comment:
Is this where the patch version would go? Or does that part of the tagging happen separately?

Reply:
We haven't put the patch part. It is only the release part.

Reply:
Gotcha. LGTM